packages/zoe/test/swingsetTests/zoe/vat-alice.js

```diff
@@ -540,6 +540,67 @@ const build = async (log, zoe, issuers, payments, installations, timer) => {
     await showPurseBalance(simoleanPurseP, 'aliceSimoleanPurse', log);
   };
 
+  const doShutdownAutoswap = async () => {
+    const issuerKeywordRecord = harden({
+      Central: moolaIssuer,
+      Secondary: simoleanIssuer,
+    });
+    const { publicFacet, adminFacet } = await E(zoe).startInstance(
+      installations.autoswap,
+      issuerKeywordRecord,
+    );
+    const liquidityIssuer = await E(publicFacet).getLiquidityIssuer();
+    const liquidityBrand = await E(liquidityIssuer).getBrand();
+    const liquidity = value => AmountMath.make(liquidityBrand, value);
+
+    // Alice adds liquidity
+    // 10 moola = 5 simoleans at the time of the liquidity adding
+    // aka 2 moola = 1 simolean
+    const addLiquidityProposal = harden({
+      give: { Central: moola(10n), Secondary: simoleans(5n) },
+      want: { Liquidity: liquidity(10n) },
+    });
+    const paymentKeywordRecord = harden({
+      Central: moolaPayment,
+      Secondary: simoleanPayment,
+    });
+    const addLiquidityInvitation = E(publicFacet).makeAddLiquidityInvitation();
+    const addLiqSeatP = await E(zoe).offer(
+      addLiquidityInvitation,
+      addLiquidityProposal,
+      paymentKeywordRecord,
+    );
+
+    console.log(await E(addLiqSeatP).getOfferResult());
+
+    const liquidityPayout = await E(addLiqSeatP).getPayout('Liquidity');
+
+    const liquidityTokenPurseP = E(liquidityIssuer).makeEmptyPurse();
+    await E(liquidityTokenPurseP).deposit(liquidityPayout);
```

Comment:
What's the purpose of the stuff between starting the contract and shutting it down?

Reply:
copied from a previous test. Not adding anything here.